.env

```
@@ -0,0 +1,2 @@
+VITE_KAKAO_APP_KEY=46c771e694eeaa9c5b933c24b21cb596
+VITE_KAKAO_TEMPLATE_KEY=126762
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)